### PR TITLE
Fix for NoneType is not iterable

### PR DESCRIPTION
**Pull request by jakub-safetycli**
## Type of Change

- [X] Bug fix
- [ ] New feature
- [ ] Documentation update
- [ ] Refactor
- [ ] Other (please describe):

## Testing

- [X] Tested by running `safety scan`
- [ ] Tests added or updated
- [ ] No tests required

## Checklist

- [ ] Code is well-documented
- [ ] Changelog is updated (if needed)
- [ ] No sensitive information (e.g., keys, credentials) is included in the code
- [ ] All PR feedback is addressed

### CODE REVIEW

**Comment by jakub-safetycli** [comment]
@SafetyQuincyF did you run into the GitHub action build issues with `build-binaries` step?

**Comment by SafetyQuincyF** [comment]
> @SafetyQuincyF did you run into the GitHub action build issues with `build-binaries` step?

I did not.

File: setup.cfg [hunk 1]
```diff
@@ -32,7 +32,7 @@ classifiers =
 [options]
 zip_safe = False
 include_package_data = True
-packages = safety, safety.formatters,safety.formatters.schemas, safety.alerts, safety.auth, safety.scan, safety.scan.finder, safety.scan.ecosystems, safety.scan.ecosystems.python, safety.alerts.templates, safety.templates
+packages = safety, safety.formatters,safety.formatters.schemas, safety.alerts, safety.auth, safety.scan, safety.scan.finder, safety.scan.ecosystems, safety.scan.ecosystems.python, safety.alerts.templates, safety.templates, safety.scan.fun_mode
 python_requires = >=3.7
```
[the remark this answers is not in the document]
Comment (SafetyQuincyF) [reply setup.cfg:36]: @jakub-safetycli

**Review by SafetyQuincyF — APPROVED** [review]

File: setup.cfg (hunk 1 above)
[the remark this answers is not in the document]
Comment (jakub-safetycli) [reply setup.cfg]: Addressed and submitted new revision.

**Pull request merged**